iothub/service/src/RegistryManager.cs

```diff
@@ -46,6 +53,66 @@ public static RegistryManager CreateFromConnectionString(string connectionString
             return new HttpRegistryManager(iotHubConnectionString, transportSettings);
         }
 
+#if !NET451
+
+        /// <summary>
+        /// Creates an instance of <see cref="RegistryManager"/>.
+        /// </summary>
+        /// <param name="hostName">IoT hub host name.</param>
+        /// <param name="credential">AAD <see cref="TokenCredential"/> to authenticate with IoT hub.</param>
+        /// <param name="transportSettings">The HTTP transport settings.</param>
+        /// <returns>An instance of <see cref="RegistryManager"/>.</returns>
+        public static RegistryManager Create(
+            string hostName,
+            TokenCredential credential,
+            HttpTransportSettings transportSettings = default)
+        {
+            if (string.IsNullOrEmpty(hostName))
+            {
+                throw new ArgumentNullException($"{nameof(hostName)} is null or empty");
+            }
+
+            if (credential == null)
+            {
+                throw new ArgumentNullException($"{nameof(credential)} is null");
+            }
+
+            TlsVersions.Instance.SetLegacyAcceptableVersions();
```

Comment:
Why call this? It does nothing when compiled with .NET 4.5.1, and this code block is in a !NET451.

Reply:
I had not seen the implementation so did not realize that. Will remove it.